release_images.yml

```diff
@@ -2,182 +2,45 @@
-      python_versions: ["py37"]
-      os_version: "ubuntu18.04"
-      neuron_sdk_version: "sdk1.19.1"
-      example: False        # [Default: False] Set to True to denote that this image is an Example image
-      disable_sm_tag: True # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
-      # to images being published.
-      force_release: False   # [Default: False] Set to True to force images to be published even if the same image
-      # has already been published. Re-released image will have minor version incremented by 1.
-  4:
-    framework: "pytorch"
-    version: "1.11.0"
-    arch_type: "x86"
-    training:
-      device_types: [ "hpu" ]
-      python_versions: [ "py38" ]
-      synapseai_version: "1.5.0"
-      os_version: "ubuntu20.04"
-      example: False
-      disable_sm_tag: True  # [Default: False] This option is not used by Example images
-      force_release: False
-  5:
-    framework: "pytorch"
-    version: "1.11.0"
-    arch_type: "x86"
-    training:
-      device_types: [ "hpu" ]
-      python_versions: [ "py38" ]
-      synapseai_version: "1.5.0"
-      os_version: "ubuntu20.04"
-      example: True
-      disable_sm_tag: True  # [Default: False] This option is not used by Example images
-      force_release: False
-  6:
-    framework: "tensorflow"
-    version: "2.9.1"
-    arch_type: "x86"
-    training:
-      device_types: [ "hpu" ]
-      python_versions: [ "py38" ]
-      synapseai_version: "1.5.0"
-      os_version: "ubuntu20.04"
-      example: False
-      disable_sm_tag: True  # [Default: False] This option is not used by Example images
-      force_release: False
-  7:
-    framework: "tensorflow"
-    version: "2.9.1"
-    arch_type: "x86"
-    training:
-      device_types: [ "hpu" ]
-      python_versions: [ "py38" ]
-      synapseai_version: "1.5.0"
-      os_version: "ubuntu20.04"
-      example: True
-      disable_sm_tag: True  # [Default: False] This option is not used by Example images
-      force_release: False
-  8:
-    framework: "huggingface_tensorflow_trcomp"
-    version: "2.6.3"
-    hf_transformers: "4.17.0"
-    arch_type: "x86"
-    training:
-      device_types: [ "gpu" ]
-      python_versions: [ "py38" ]
-      os_version: "ubuntu20.04"
-      cuda_version: "cu112"
-      example: False
-      disable_sm_tag: False  # [Default: False] This option is not used by Example images
-      force_release: False
-  9:
     framework: "pytorch"
     version: "1.11.0"
     arch_type: "x86"
     customer_type: "sagemaker"
     training:
-      device_types: [ "cpu", "gpu" ]
-      python_versions: [ "py38" ]
-      cuda_version: "cu113"
-      os_version: "ubuntu20.04"
-      example: False
-      disable_sm_tag: False
-      force_release: False
-  10:
-    framework: "pytorch"
-    version: "1.12.0"
-    arch_type: "x86"
-    customer_type: "sagemaker"
-    training:
-      device_types: [ "cpu", "gpu" ]
+      device_types: ["gpu"]
```

Comment:
Just GPU?

Reply:
Yes, confirmed it with @viskaria.(Ticket- https://t.corp.amazon.com/P68241325/overview)